src/DiabloInterface/Gui/MainWindow.cs

```diff
@@ -177,6 +178,35 @@ void DebugMenuItemOnClick(object sender, EventArgs e)
             debugWindow.Show();
         }
 
+        void DifficultyNormalToolStripMenuItemOnClick(object sender, EventArgs e)
+        {
+            UnheckDifficultyMenuItems();
```

Comment:
Oh that's an unfortunate typo!
Hmm... Not sure about the `...MenuItemOnClick` methods because they are bound to different buttons.
Tell once you've gotten around to testing it, because you might have some ideas about how it works once you see it in action!